engine/src/defaults/plugin.cpp

```diff
@@ -39,9 +41,13 @@ void cubos::engine::defaultsPlugin(Cubos& cubos)
     cubos.plugin(solverPlugin);
     cubos.plugin(renderDefaultsPlugin);
 
+    cubos.plugin(canvasPlugin);
+    cubos.plugin(colorRectPlugin);
+
     cubos.plugin(gizmosPlugin);
     cubos.plugin(imguiPlugin);
 
+    cubos.tag(uiDrawTag).after(toneMappingTag);
     cubos.tag(gizmosDrawTag).after(toneMappingTag);
     cubos.tag(imguiEndTag).after(gizmosDrawTag);
```

Comment:
do we want UI to be drawn after gizmos? I think it would make sense for it to be drawn after world gizmos, but not space or view gizmos

Reply:
Let's draw it after gizmos entirely, currently world/space/view gizmos are joined together in a single system. We would have to split it into two systems for that to work. If you can, open an issue for this so that we can change it later.